src/auto-evo/RunResults.cs

```diff
@@ -182,21 +208,6 @@ public void AddNewSpecies(Species species, IEnumerable<KeyValuePair<Patch, long>
         }
     }
 
-    public void AddSplitResultForSpecies(Species species, Species splitSpecies, List<Patch> patchesToConvert)
-    {
-        if (patchesToConvert == null || patchesToConvert.Count < 1)
-            throw new ArgumentException("split patches is missing", nameof(patchesToConvert));
-
-        MakeSureResultExistsForSpecies(species);
-        MakeSureResultExistsForSpecies(splitSpecies);
-
-        results[species].SplitOff = splitSpecies;
-        results[species].SplitOffPatches = patchesToConvert;
```

Comment:
Looking at this more, there's a ton of code that depends on SplitOffPatches. So I'm highly super questioning that things are very broken related to that due to the new auto-evo and this situation needs to be fixed. I don't think I have time to fix the 23 errors that come from commeting out the split off and split off patches variables.